src/panels/lovelace/cards/energy/hui-energy-compare-card.ts

```diff
@@ -0,0 +1,113 @@
+import { differenceInDays, endOfDay } from "date-fns";
```

Comment:
```suggestion
import { differenceInDays, endOfDay } from "date-fns/esm";
```

Reply:
It should already use that as it has:
```
  "main": "index.js",
  "module": "esm/index.js",
```
in `package.json`?